core-feature-pack/src/main/resources/content/bin/standalone.sh

```diff
@@ -262,18 +262,29 @@ if [ "$PRESERVE_JAVA_OPTS" != "true" ]; then
         fi
     fi
 
+    # Set flag if JVM is modular
+    setModularJdk
+
     if [ "$GC_LOG" = "true" ]; then
         # Enable rotating GC logs if the JVM supports it and GC logs are not already enabled
-        NO_GC_LOG_ROTATE=`echo $JAVA_OPTS | $GREP "\-verbose:gc"`
+        mkdir -p $JBOSS_LOG_DIR
+        NO_GC_LOG_ROTATE=`echo $JAVA_OPTS | $GREP "\-Xlog\:\?gc"`
```

Comment:
This would need to check for both `-Xlog:gc` and `-Xloggc`.

Reply:
@jamezp it should be checking both of them
$ echo "-Xlog:gc" | grep "\-Xlog\:\?gc"
-Xlog:gc
$ echo "-Xloggc" | grep "\-Xlog\:\?gc"
-Xloggc

Reply:
Yup sorry I misread the regex.